service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DocumentReferenceToNarrativeStatementMapperTest.java

```diff
@@ -77,7 +84,11 @@ public void setUp() throws IOException {
         final Bundle bundle = new FhirParseService().parseResource(bundleInput, Bundle.class);
         messageContext = new MessageContext(randomIdGeneratorService);
         messageContext.initialize(bundle);
-        mapper = new DocumentReferenceToNarrativeStatementMapper(messageContext);
+
+        lenient().when(supportedContentTypes.isContentTypeSupported(eq("text/richtext"))).thenReturn(true);
+        lenient().when(supportedContentTypes.isContentTypeSupported(eq("application/octet-stream"))).thenReturn(false);
```

Comment:
No need for `eq()` etc when every parameter is given
```suggestion
        lenient().when(supportedContentTypes.isContentTypeSupported("text/richtext")).thenReturn(true);
        lenient().when(supportedContentTypes.isContentTypeSupported("application/octet-stream")).thenReturn(false);
```